### DISCUSSION TITLE
Can code annotations be reused in the same code block?

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#6300] [Q&A] [answered]
I've noticed a rather unfortunate behaviour of code annotations, where if you define the same number multiple times in the code block, the annotation will only be applied to the last entry of the same number, making any previous one be empty.

As an example:
````markdown
```yaml
option:
  option1: lorem # (1)
  option2: ipsum # (2)
  option3: sit # (1)
```

  1. Some description that will only appear in the second entry of the code annotation. First entry will be empty
  2. Another description.
````
In this example will the annotation for `option1: lorem` be empty, not even having the rotation animation on click.
Meanwhile the second annotation is fine and the third one is using the 1st one.

I'm not sure if this is considered intentional behaviour or actually a bug, which is why I ask here instead of filing a bug report, as it isn't clear to me.

Preferably would I like the code annotations to allow being reused multiple times, as it would reduce the duplicate text you would otherwise need to have here...

**Comment by squidfunk** [comment, marked as answer]
Duplicate https://github.com/squidfunk/mkdocs-material/discussions/3832#discussioncomment-2616875.

**Reply by squidfunk** [reply]
[As mentioned in our documentation](https://squidfunk.github.io/mkdocs-material/contributing/#before-asking-a-question), it is always a good idea to use the search before posting a new discussion or creating a new issue. It saves you and us time, as we don't have to answer the same questions again and again.

**Reply by Andre601** [reply]
Well, my search didn't show anything. Probs not good enough search terms or a glitch in GitHub's search then.

So please don't accuse me of not doing thing when I actually did them. Thanks.

**Reply by squidfunk** [reply]
I didn't accuse you ☺️ I just pointed out that it is a good idea for the reasons stated. Took me 1m 😉
